src/haddock/clis/cli_mpi.py

```diff
@@ -0,0 +1,91 @@
+"""
+MPI-wrapper for executing pickled CNSJobs created by libmpi.
+
+This was developed for use of lbimpi but it might be useful in some specific
+ scenario as a cli.
+
+For more information please refer to the README.md in the examples folder.
+
+Usage:
+    haddock3-mpi -h
+    haddock3-mpi tasks.pkl
+"""
+
+import argparse
+import pickle
+import sys
+
+
+try:
+    from mpi4py import MPI
```

Comment:
Can we have ?
```python
from mpi4py.MPI import COMM_WORLD as COMM
```

and avoid the `COMM` definition below?

Reply:
why?